integration_tests/pages/apply/prisonNumberPage.ts

```diff
@@ -0,0 +1,12 @@
+import Page from '../page'
+
+export default class PrisonNumberPage extends Page {
```

Comment:
minor but maybe this would be better named `FindByPrisonNumberPage`?

Reply:
good spot, changed